### DISCUSSION TITLE
React tsParticle component used solely in one section of the website

### DISCUSSION THREAD

**Discussion by jeremiahhaastrup** [matteobruni/tsparticles#4801] [General]
Hi, I've been trying to see whether it would be possible to have the tsParticles shown solely in one section of a website and allow for a clear background when scrolling to other sections, rather than the background remaining stationary. Any suggestions?

**Comment by matteobruni** [comment]
https://www.reddit.com/r/tsParticles/comments/w8luvb/why_my_particles_are_full_screen_i_want_them_to/

**Comment by jeremiahhaastrup** [comment]
Hi Matteo, thanks for the reply. I figured out that making the position relative for each section resolves the issue.
